sdk/storage/Azure.Storage.Files.Shares/perf/README.md

```diff
@@ -0,0 +1,15 @@
+# Azure Storage Files Shares performance tests
```

Comment:
You'll also need to add an exception to the [doc config](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/.docsettings.yml) to avoid this getting flagged by the build analyzer.